source/includes/_clients.md

```diff
@@ -94,6 +141,18 @@ client-id | <span class="label label-required">Required</span>
 
 <span class="label label-info">Account Method</span>
 
+> Get a Client
+
+```php
+<?php
+use BrightLocal\Api;
+$id = 1;
+
+$api = new Api('<INSERT_API_KEY>', '<INSERT_API_SECRET>');
+$client = $api->post('v1/clients-and-locations/clients/'. $id.);
```

Comment:
Additional "." after $id. Also, perhaps call the variable $clientId rather than $id to emphasise what it is.